janusgraph-solr/src/main/java/org/janusgraph/diskstorage/solr/SolrIndex.java

```diff
@@ -257,6 +265,31 @@ public void register(String store, String key, KeyInformation information, BaseT
             }
         }
         //Since all data types must be defined in the schema.xml, pre-registering a type does not work
+        //But we check Analyse feature
```

Comment:
Is this check here necessary since it looks like errors would be handled in your `customTokenize` method?